tests/wpt/mozilla/meta/mozilla/service-workers/service-worker-registration.https.html.ini

```diff
@@ -0,0 +1,6 @@
+[service-worker-registration.https.html]
+  type: testharness
+  [Test: Asserts Active Service Worker and its Registration]
+      expected: FAIL
+  [Test: Active Service Worker ScriptURL property]
+      expected: FAIL
```

Comment:
Note, see https://github.com/servo/servo/issues/26378 for why this fails and how it will be fixed. I've written some tests making similar assertions using the "installing worker".